client/src/components/SearchResultList/SearchResultList.js

```diff
@@ -0,0 +1,5 @@
+import styled from "@emotion/styled";
+
+const SearchResultList = styled.nav``;
+
+export default SearchResultList;
```

Comment:
Why are you exporting an unstyled nav as styled Component?
Isn't it simpler just to use a nav in your list?

Reply:
I love complicating my life, and well... the name is cool! thanks!